src/anyp/Uri.cc

```diff
@@ -1049,4 +1102,3 @@ AnyP::Uri::cleanup(const char *uri)
     assert(cleanedUri);
     return cleanedUri;
 }
-
```

Comment:
Please remove this out-of-scope change (probably caused by GitHub -- I hope that we will stop seeing these after #591).

Reply:
Ack. Will do.

Reply:
Done in 7fc9da84b4ca3f57fa6190f369098fdd6a9313b4